docs/registry/upload/_index.md

```diff
@@ -137,7 +137,7 @@ If you later wish to make your module public, you can use the [`viam module upda
     {
       "module_id": "acme:my-module",
       "visibility": "public",
-      "url": "https://github.com/acme-co-example/my-module",
```

Comment:
I think there are two more occurrences of this link across the docs, should we change those too?

Reply:
Thanks, good eye -- updated them also!